domainhunter.py

```diff
@@ -18,6 +18,7 @@
 import sys
 import urlparse
```

Comment:
Modify line to correct import the Python3 library `from urllib.parse import urlparse`

Reply:
syntax updated

Reply:
reqs updated, getpass is part of the python core.